.gitignore

```diff
@@ -29,3 +29,7 @@ tmp/
 
 # test mysql server data
 mysqldata/
+
+# generated lock files
+package-lock.json
+yarn.lock
```

Comment:
Nit: github wants us to put a newline at the end of the file
![image](https://user-images.githubusercontent.com/618009/106081045-dacf7500-60dd-11eb-8017-9e041a75c9e4.png)

Reply:
Actually we need `yarn.lock` to ensure consistency in the JS packages for builds. I'm not sure about `package-lock.json`.

@noahtalerman can you please revert this change to the gitignore and include `yarn.lock`? If git also wants to add changes to a `package-lock.json` I'm happy to look at that with you.